python/src/deltachat/account.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import print_function
+import threading
+import requests
+from . import capi
+import deltachat
+from .capi import ffi
+
+
+class Account:
+    def __init__(self, db_path, logcallback=None):
+        self.dc_context = ctx = capi.lib.dc_context_new(
+                                  capi.lib.py_dc_callback,
+                                  capi.ffi.NULL, capi.ffi.NULL)
+        capi.lib.dc_open(ctx, db_path, capi.ffi.NULL)
+        self._logcallback = logcallback
+
+    def set_config(self, **kwargs):
+        for name, value in kwargs.items():
```

Comment:
will all names always be valid python identifiers?  The `**kwargs` would stop deltachat from using e.g. `account-name` as a config name.